test/AlgebraicGeometry/Surfaces/EnriquesAut.jl

```diff
@@ -0,0 +1,46 @@
+  SX = lattice(V, B)
+  
+  B = matrix(QQ, 10, 26 ,[-4, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, 2, 0, -3, 2, 0, 0, 0, -1, 1, 0, 0, 1, -1, 0, 0, 0, 0, -1, 1, 0, 0, 0, -1, 1, 0, 0, 0, 0, 2, -4, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, -1, 1, 0, 2, 2, 0, 2, -4, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -2, 0, -1, 1, -2, 0, 0, 0, 2, -3, 1, -1, 1, -1, 0, -1, 1, 0, 0, 0, 0, 1, 1, 0, -1, 1, 3, -1, -1, -2, -1, 0, 0, 0, 0, 0, -1, 2, -1, 1, 0, 1, -1, 0, 0, 0, 0, -1, -2, 0, 1, -2, -2, 1, 1, 1, 1, 0, 0, 0, 0, 1, -1, 0, -1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, -1, 1, 1, 1, 0, 0, -1, 0, 0, 0, 1, -1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, -2, -1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, -2, 0, 0, 0, 0, 0, -1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, -2, 0, -1, 1, 1, 1, 1, 0, 0, 0, 0]);
+  SY = lattice(V, B)
+  
+  weyl = ZZ[2   -3   2   2   -5   2   1   1   0   0   2   -2   -1   -1   -3   2   -3   2   -4   -1   -3   0   1   0   -2   1]
+  
+  Y = Oscar.EnriquesBorcherdsCtx(SY, SX, L26, weyl)
+  autY, chambersY, ratsY = borcherds_method(Y)
+  @test length(chambersY)==1
+  C = only(chambersY)
+  @test length(rays(C))==6457
+  @test length(Oscar.isotropic_rays(C)) == 247
+  @test length(walls(C)) == 40
+  @test length(ratsY) == 2
+  @test mass(Y) == 1//60
+  @test Oscar.root_invariant(Y)[2][1] == [(:A, 4), (:A, 5)]
+  @test length(Oscar.splitting_roots_mod2(Y)) == 25
+  @test rays(C) == Oscar.initial_rays(Y)
+  @test walls(C) == Oscar.initial_walls(Y)
+  @test length(Oscar.initial_automorphisms(Y)) == 1440
+  
+  v = Oscar.isotropic_rays(C)[1]
+  L = lattice(rational_span(SY))
+  Oscar.frame_lattice(L,QQ.(v))
+  @test Oscar.chamber_invariants(Y)[1] == 1440
+  
+  fbar = discriminant_group(Y.SY)(1//2*(v*basis_matrix(Y.SY))[1,:])
+  @test reducible_fibers(Y, fbar) == ([(:A, 3), (:A, 1), (:A, 3)],[])
```

Comment:
Is it clear that this order will only be respected in the future, if we do any change in the ordering of certain outputs for root lattice recognition or else ?